src/nunit.analyzers.tests/TestCaseUsage/TestCaseUsageAnalyzerTests.cs

```diff
@@ -824,6 +835,18 @@ public void TestWithGenericParameter<T>(T arg1) { }
     }");
             RoslynAssert.Valid(this.analyzer, testCode);
         }
+
+        [TestCaseSource(nameof(SpecialConversions_NET6))]
```

Comment:
This code is inside an #if NUNIT4 conditional, but works fine with NUNIT3.
I have moved the method.